src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckPolicyOverrideCollector.java

```diff
@@ -91,7 +93,7 @@ public BlackDuckPolicyOverrideCollector(final JsonExtractor jsonExtractor, final
             ComponentItemPriority priority = mapSeverityToPriority(policyItem.getSeverity());
             Set<LinkableItem> attributeSet = new LinkedHashSet<>();
             nameItem.ifPresent(attributeSet::add);
-            attributeSet.addAll(createPolicyLinkableItems(policyItem));
+            attributeSet.addAll(createPolicyLinkableItems(policyItem, bomComponentVersionUrl.orElse("")));
```

Comment:
I think down the line, this bomComponentVersionUrl trickles to the bucket. How does it handle empty strings? If it does an `isBlank` check, I would be more comfortable with null here so that we don't potentially have the side affect of an empty string as a key in the bucket if it ever changes.